template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/IdentityServiceExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.Service;
+using Microsoft.AspNetCore.Identity.Service.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.Service.IntegratedWebClient;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Company.WebApplication1.Identity.Data;
+using Company.WebApplication1.Identity;
+using Company.WebApplication1.Identity.Models;
+using Company.WebApplication1.Identity.Services;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class IdentityServiceExtensions
+    {
+        public static IServiceCollection AddIdentityService(this IServiceCollection services, IConfiguration config)
```

Comment:
Refactored these methods. For preview1 some code will be in an Extensions folder that we can move into the framework for preview2.  https://github.com/aspnet/IdentityService/issues/84